### ISSUE TITLE
Add array-type operators

### ISSUE THREAD

**Issue by michalgm** [feathersjs-ecosystem/feathers-knex#235]
It would be nice to have operators for postgres array-type fields:
https://www.postgresql.org/docs/9.0/functions-array.html
Incoming PR to add:
- contains (`@>`)
- contained_by (`<@`)
- overlap (`&&`)

**Comment by MarcGodard** [comment]
I agree to this, this has been causing a lot of issues. I have been storing arrays into json fields, as JSON.stringify, but I cannot query these. Been trying to figure out a way around this and other than storing it as text, then managing it... I cannot seem to find an easy way.
